app/queries/effort_query.rb

```diff
@@ -271,7 +271,7 @@ def self.over_segment_subquery(segment, existing_scope)
               laps_required = 0
                   and ss.effort_id in (select effort_id from effort_stopped)
           ) or (
-              laps_required > 0 and laps_finished >= laps_required
+              laps_required > 0 and coalesce(laps_finished, 0) >= laps_required
```

Comment:
This field was returning `null` for unfinished efforts, resulting in empty columns in the UI.